go.mod

```diff
@@ -3,20 +3,24 @@ module github.com/kong/goks
 go 1.19
 
 require (
+	github.com/gofrs/uuid v4.3.0+incompatible
 	github.com/google/uuid v1.3.0
+	github.com/kong/atc-router v0.0.0-20220914144203-b1b8915e52e7
 	github.com/layeh/gopher-json v0.0.0-20201124131017-552bb3c4c3bf
 	github.com/santhosh-tekuri/jsonschema/v5 v5.0.0
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.8.0
 	github.com/yuin/gluare v0.0.0-20170607022532-d7c94f1a80ed
 	github.com/yuin/gopher-lua v0.0.0-20210529063254-f4c35e4016d9
 )
 
 require (
-	github.com/davecgh/go-spew v1.1.0 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
 replace github.com/yuin/gopher-lua => github.com/hbagdi/gopher-lua v0.0.0-20211129210354-3e4a277fb892
 
 replace github.com/layeh/gopher-json => github.com/hbagdi/gopher-json v0.0.0-20220325165250-3030ea88774d
+
+replace github.com/kong/atc-router => ../atc-router/lib/go-atc-router
```

Comment:
this of course "works on my machine" :rofl: 

it seems there's no way to create a go module within a non-go repository, so the binding will have to have it's own repo.

Reply:
fixed